<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameIntOption</class>
 <widget class="QWidget" name="GameIntOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>34</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="lText">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sbValue">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="minimum">
      <number>-99999</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bSet">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>122</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
